secret-provider/src/main/scala/io/lenses/connect/secrets/config/AWSProviderSettings.scala

```diff
@@ -39,6 +40,8 @@ object AWSProviderSettings {
     val authMode =
       getAuthenticationMethod(configs.getString(AWSProviderConfig.AUTH_METHOD))
 
+    val altRegion = configs.getString("aws.cross.account.region")
```

Comment:
Instead of using the string `aws.cross.account.region` here, it would be better to refer to the constant you defined `AWS_CROSS_ACCOUNT_REGION`